pkg/dbutil/table_test.go

```diff
@@ -169,5 +169,5 @@ func (*testDBSuite) TestSchemacmpEncode(c *C) {
 	c.Assert(err, IsNil)
 
 	table := schemacmp.Encode(tableInfo)
-	c.Assert(table.String(), Equals, "CREATE TABLE `tbl`(`id` INT(24) NOT NULL, PRIMARY KEY (`id`)) CHARSET UTF8MB4 COLLATE UTF8MB4_BIN")
+	c.Assert(table.String(), Equals, "CREATE TABLE `tbl`(`id` INT(24), PRIMARY KEY (`id`)) CHARSET UTF8MB4 COLLATE UTF8MB4_BIN")
```

Comment:
i am not sure why in old test when create SQL is

```sql
CREATE TABLE `test`.`atest` (`id` int(24), primary key(`id`))
```
will return a 

```sql
CREATE TABLE `tbl`(`id` INT(24) NOT NULL, PRIMARY KEY (`id`)) CHARSET UTF8MB4 COLLATE UTF8MB4_BI
```
after I update tidb , the return SQL looks more normal